app/views/orders/show.html.haml

```diff
@@ -0,0 +1,22 @@
+- provide(:title, "My Order")
+
+#site-main
+%h1 Your Order
+
+.container-menu
+  #site-menu
+    #menu
+      %ul.product-list
+        - @order_products.each do |order_product|
+          %li.dish-item
+            %div
+              %p.text-title= order_product.product.title
+              %p.text-description= "Quantity: #{order_product.quantity}"
+              %p.text-description.dish-price= "Price per Unit: $#{order_product.product.price}"
+              %p.text-price.dish-price= "Total per Dish Type: $#{order_product.product.price*order_product.quantity}"
+              = link_to 'Remove from Cart', remove_from_order_order_path(@order, order_product.product), method: :delete, data: { confirm: 'Are you sure you want to remove this item from your cart?' }
+    %h1
+      .text-total-order
+        Total Order:
+        = number_to_currency(calculate_order_total(@order), precision: 2, format: '%n') + "$"
+                    
```

Comment:
<img width="89" alt="image" src="https://github.com/andreeap00/foodie-app/assets/19929642/97581eb0-423d-41b5-8b47-c5eef45616ee">
the newline still seems to not be there :( same happens on the next files